testgrid/config/config.yaml

```diff
@@ -1856,6 +1858,8 @@ dashboards:
     test_group_name: ci-kubernetes-e2e-kops-aws-updown
   - name: kops-aws-weave
     test_group_name: ci-kubernetes-e2e-kops-aws-weave
+  - name: kops-gce
+    test_group_name: ci-kubernetes-e2e-kops-gce
```

Comment:
It's on the google-aws dashboard for now, because that is where all the kops jobs are.  Once this goes green we should probably either rename google-aws and/or create a new `kops` dashboard.

Reply:
how about rename it to google-kops?

Reply:
Can we have two jobs: google-kops-aws and google-kops-gce? We can create new kops dashboard later.